docs/auth.md

```diff
@@ -1,44 +1,64 @@
 # Auth service
 
-You can use `AutomationCloudAuthService` to authenticate requests made to automation cloud, It will try to parse the header injected by Gateway, then try to forward the auth header to s-api if the header is not present(for backward compatibility).
-If you were using `ForwardRequestHeaderAuthService`, use `AutomationCloudAuthService` instead (as described here.)
+You can use `AutomationCloudAuthService` to authenticate requests made to automation cloud, It will decode anf verify the auth header injected by Gateway - header name is configured as `process.env.AC_AUTH_HEADER_NAME`) if presented, or it will forward the auth header(`authorization`) to s-api if not presented(for backward compatibility). if none of them is provided, it will not check any auth. for this reason, it is highly recommended for you to add additional `this.acCotext.checkAuthenticated()` from your router's middleware.
 
+> AuthService is deprecated and replaced by `RequestAuthService`.
+> If you were using `ForwardRequestHeaderAuthService`, bind `RequestAuthService` to `AutomationCloudAuthService` instead as described here.
 
 ## Example
 ```ts
 // src/main/routes/my-router.ts
 import {
     Application,
-    AuthService,
+    RequestAuthService, // previously AuthService
     AutomationCloudAuthService,
 } from '@ubio/framework';
 
 export class App extends Application {
 
     constructor() {
         super();
-        this.container.bind(AuthService).to(AutomationCloudAuthService);
+        this.container.bind(RequestAuthService).to(AutomationCloudAuthService);
```

Comment:
Didn't we bind this to base app already? 🤔  Probs the entire section can be removed.

Reply:
(it was sort of the purpose for apps to not concern themselves with how auth works, so if you want to include the docs on how it works internally, I suggest you move this to the bottom and label with "Internal implementation" heading)

Reply:
Oh my bad!!! will remove it (bad editing)